tests/convolve.py

```diff
@@ -172,6 +177,7 @@ def sameMaskPlaneDicts(maskedImageA, maskedImageB):
     return True
 
 class ConvolveTestCase(utilsTests.TestCase):
+    @unittest.skipIf(dataDir is None, "afwdata not setup")
```

Comment:
I would expect that is setUp is skipped that all the tests in the class would be skipped so there would be no need to try to skip other functions explicitly.